brightray/browser/browser_main_parts.cc

```diff
@@ -156,11 +163,41 @@ BrowserMainParts::BrowserMainParts() {
 BrowserMainParts::~BrowserMainParts() {
 }
 
+void OverrideWinAppLogsPath() {
+  int status;
+  std::string appName = GetApplicationName();
+  std::string logPath = "%HOMEDRIVE%%HOMEPATH%\\AppData\\Roaming\\";
+  std::string appLogPath = logPath + appName + "\\logs";
+
+  status = mkdir(appLogPath.c_str(), S_IRWXU | S_IRGRP | S_IROTH);
```

Comment:
The `App::GetPath` API (which calls `PathService::Get`) automatically creates the dir so there is no need to create it here.